bi-connectors/PowerBIConnector/OpenSearchProject.pq

```diff
@@ -28,7 +28,7 @@ OpenSearchProjectType = type function (
             Documentation.FieldDescription = "Use SSL",
             Documentation.AllowedValues = { true, false }
         ]),
-        HostnameVerification as (type logical meta [
+        optional HostnameVerification as (type logical meta [
```

Comment:
Have you tested opening PBIDS files generated with the latest connector to see if they work if the optional parameter is not given a value?
There could also be issues with TestConnection as it is expecting HostnameValidation to be set. You need to test with Gateway to confirm.